yGavrilovets/01_HTML/HW1_additionalTask.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Additional_Task</TITLE>
+	</HEAD>
+	<BODY>
+		<H1 align="center">Заголовок первого уровня</H1>
+		<H2>Предварительно форматированный текст:</H2>
+		<pre>x | o |
+‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐
+  | х |
+‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐
+o | x | o</pre>
+		<HR>
+		<H2>Тег address</H2>
+		
+		<P class="adr"><i><span class="street-address">Kuprevicha</span>, <span class="house-number">1/1</span>, <span class="postal-code">220141</span>, <span class="region">Minsk</span>, <span class="country-name">Belarus</span>. <span class="tel">+375.17.2686911</span></i></P>
+		<HR>
+		<H2>Семантические и функциональные строчные теги</H2>
+		<P>Пожелания автора:<br><del style="color:Tomato;">Данный текст убрать из монографии</del> и заменить его на <ins style="color:DodgerBlue;">вновь помещённый текст.</ins></P>
+		<HR>
+		<H2>Работа со строчными и блоковыми цитатами</H2>
+		<P>Цитата короткая:<br><cite>И всё-таки она вертится!</cite></P>
+		<P>Цитата занимающая несколько предложений:</P>
+		<blockquote>О сколько нам открытий чудных <br>Готовят просвещенья дух <br>И опыт, сын ошибок трудных, <br>И гений, парадоксов друг, <br>И случай, бог изобретатель...</blockquote>
+		<HR>
+		<H2>Семантические и функциональные строчные теги</H2>
+		<P><strong>Тэг 	&lt;STRONG> делает данный фрагмент текста более важным.</strong> <em>Тэг 	&lt;em> менее важным.</em> В то время как теги 	&lt;b> и  &lt;i> делают текст просто <b>полужирным</b> и <i>курсивом</i> соответственно.</P>
+		<HR>
+		<H2>Теги форматирования машинного языка</H2>
+		<P>При выполнении программного кода на языке PERL:</P>
+		<pre><code>#!/usr/local/bin/perl
+print "Content‐type: text/html\n\n";
+print "&lt;H1>Hello World&lt;/H1>\n";</code></pre>
+		<P>Программа выведет:</P>
+		<code>Hello, World!</code>
+		<P>Однако в браузере у нас будет отображаться результат выполнения этой программы:</P>
+		<H1>Hello, World!</H1>
+		<HR>
+		<H2>Комментарии и литералы(мнемоники) в HTML</H2>
+		<pre>&lt;!-- это комментарий --></pre>
+		<pre>		&lt;!-- и это также комментарий,
+		занимающий более одной строки --></pre>
+		<HR>
+		<H2>Теги над­ и подстрочных символов</H2>
+		<H3 align="center">Магазин работает:</H3>
+		<P align="center"center>c 08<sup><u>30</u></sup> до 18<sup><u>30</u></sup><br>обед с 13<sup><u>00</u></sup> до 14<sup><u>00</u></sup><br>выходной: <u>суббота</u>, <u>воскресенье</u></P>
+		<HR>
+		<H2>Вот такой текст</H2>
+		<H1><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></H1>
+		<HR>
+		<H2>Акроним и курсив</H2>
+		<P>&Tau;&Phi;: Y(<i>x</i>)=e<sup><i>-i&pi;x</i></sup>+log<sub>x</sub>4</P>
+		<!--ЁБАНЫЕ ФООООРМУЛЫ!!!!!!!!-->
```

Comment:
Could you please remove this line

Reply:
Ohhh((( Sorry for this, i was sure, i've deleted all comments. it was for intermediate stage of homework. there won't be such situations any more